crates/gen/src/tables/type_def.rs

```diff
@@ -118,6 +118,20 @@ impl TypeDef {
         })
     }
 
+    pub fn is_public_composable(&self) -> bool {
+        self.attributes().any(|attribute| {
+            if attribute.name() == "ComposableAttribute" {
+                for arg in attribute.args() {
+                    if let (_, ConstantValue::I32(2)) = arg {
+                        return true;
+                    }
+                }
+            }
+
+            false
```

Comment:
```suggestion
            attribute.name() == "ComposableAttribute"
                && attribute
                    .args()
                    .iter()
                    .any(|arg| matches!(arg, (_, ConstantValue::I32(2))))
```
Alternatively you could destructure the tuple in the closure parameters.